### PR TITLE
Fixes #126 : Fixed Markodwn issues and other fixes.

### LINKED ISSUE #126: Problems with Markdown jekyll rendering and other code quality issues.
* Markdown doesn't render properly.
* Buttons on the hero image don't work.
* Headings are not centrally aligned.
* Footer doesn't span the width of the entire page.
* No margin on top of footer.
* Make links FOSSASIA Red.

**I am working on it!**

### PR DESCRIPTION
**Pull request by nzec**
* Fixed markdown errors in all files.
* Made the buttons on the hero image work. (by not having a selector but actually changing the brightness of the original image)
* Centrally aligned headings.
* Made the footer span the width of the entire page.
* Change Friends Section links to Red.
* Fixes #126 

**Here is the Preview Link : https://duttaditya18.github.io/2014.fossasia.org/**

### CODE REVIEW

**Comment by nzec** [comment]
Ping @ms10398 @ParthS007

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
The red links are not there :crossed_swords: 

image

**Comment by nzec** [comment]
@0x48piraj I meant I changed this.

This

**Comment by 0x48piraj** [comment]
Please be careful. As the name you said, is a proposed task :wink: 
Please change the name, if you like. :+1:

**Review by 0x48piraj — APPROVED** [review]
Good Work !

File: _layouts/default.html [hunk 1]
```diff
@@ -8,7 +8,8 @@
         {{ content }}
       </div>
     </div>
-    </div>	
+    </div>
```
Comment (akshatnitd) [inline _layouts/default.html:11]: Why were these 2 extra `</div>` added?

Comment (nzec) [reply _layouts/default.html:11]: I added them to separate the footer from the `#wrapper`.

The wrapper was consuming the footer as well.

Comment (nzec) [reply _layouts/default.html:11]: The confusion is arising because there are 2 wrappers.
A `.wrapper` and a `#wrapper`.

**Comment by nzec** [comment]
>Please be careful. As the name you said, is a proposed task 😉
Please change the name, if you like. 👍

@0x48piraj Done!

File: _layouts/default.html (hunk 1 above)
Comment (nzec) [reply _layouts/default.html:11]: @akshatnitd Is it fine?

**Review by sayak119 — APPROVED** [review]

**Comment by nzec** [comment]
Ping @mariobehling @ms10398

**Review by harshit98 — APPROVED** [review]

**Comment by nzec** [comment]
@harshit98 Thanks!

**Pull request merged**